lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -72,9 +72,9 @@ def colorized_formatted_backtrace(colorizer=::RSpec::Core::Formatters::ConsoleCo
           end
         end
 
-        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
+        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes, skip=0)
           lines = fully_formatted_lines(failure_number, colorizer)
-          lines.join("\n") << "\n"
+          lines.drop(skip).join("\n") << "\n"
```

Comment:
Ok, I've exposed `fully_formatted_lines` instead, it allows us to do the skipping entirely within the formatter without an unnecessary join + split, or running a gsub.